.gitignore

```diff
@@ -16,4 +16,10 @@ glide.lock
 gpg/
 
 # ignore. For testing we don't want to store a specific provider.
-.terraform.lock.hcl
\ No newline at end of file
+.terraform.lock.hcl
+
+# Adding to disregard some testing framework standup. - dbarcelo
+node_modules/
+package-lock.json
+package.json
+test.js
```

Comment:
Can you elaborate on this? What testing framework outside of `go test`? This may be of benefit to the team if you found a nice tool.

Reply:
I believe this was put in because I was attempting to run GH actions locally, and ultimately test without actually running in the repo and making changes.  It was a passing try and I didn't spend too much time on it.  Ultimately it was easier to simply model out the behavior I wanted in a separate repo, got it to work the way I wanted and then copy pasta'ed to this one.

Here is the context of package-json and test.js if you are curious:
```
{
  "dependencies": {
    "@actions/core": "^1.10.1",
    "@actions/exec": "^1.1.1"
  }
}
```

```
const core = require('@actions/core');
const exec = require('@actions/exec');

async function run() {
try {
// Set the input variables (if any)
core.getInput('myInputVariable');

// Execute the steps of our workflow
await exec.exec('echo', ['Hello, World!']);
// Add more steps as necessary

// Set the output variables (if any)
// Set the output variables (if any)
core.setOutput('myOutputVariable', 'Hello from local testing!');
} catch (error) {
core.setFailed(error.message);
}
}

```

If you are curious where I modeled the actions I did it here, should be 0 Cox properties in this public repo:
https://github.com/DevOpsDave/testing-github-actions

Reply:
@LumaC0 I removed the testing stuff from my repo and took out the .gitignore entiries that hid the artifacts since nothing was actually being done with it.  Neater now.